esp/services/ws_topology/ws_topologyService.hpp

```diff
@@ -57,6 +57,9 @@ struct ReadLog
     unsigned logfields;
     unsigned columnNumDate;
     unsigned columnNumTime;
+    StringArray logFieldNames;
+    bool includeLogFieldNameLine;
```

Comment:
Good habit to start initializing new variables here within class or struct declaration.  Just new ones as part of a PR like this.